morebits.js

```diff
@@ -2136,6 +2136,13 @@ Morebits.wiki.page = function(pageName, currentAction) {
 		return ctx.pageExists;
 	};
 
+	/**
+	 * @returns {string} ISO 8601 timestamp at which the page was last loaded.
+	 */
```

Comment:
Per https://github.com/azatoth/twinkle/pull/741#issuecomment-568593321, could you add a brief comment here?  Just something noting that it's only available in a `.load` and even then only because `intoken=edit` adds `starttimestamp`, and will need to be fixed if/when moving to `meta=tokens` via `curtimestamp`.

Reply:
> something noting that it's only available in a .load 

It already says " ... timestamp at which the page was **_last loaded_**" And it's available anytime after a load, not necessarily in `.load` callback.

> only because intoken=edit adds starttimestamp, and will need to be fixed if/when moving to meta=tokens via curtimestamp.

such a comment belongs at the place where `ctx.loadTime` is being assigned (as the public interface wouldn't be affected). Added it there.